### ISSUE TITLE
Video edit name not working

### ISSUE THREAD

**Issue by nasariasl** [ampache/ampache#1236]
afterster: movies have bug !
afterster: i can edit summary,tags,video image but => i cant edit video name !!! <= when i change video name and the clicking on update button nothing happend

**Comment by wagnered** [comment]
@nasariasl,

If, by "video name" you mean  "title",  what version are you using? I
just verified that title change in 3.8.3 works correctly. Please update
to latest development version 3.8.3.

If you are referring to "Original Name" Yes, it is a bug.  I think I
fixed it in the new pull request, but the PR was put on hold because of
the development of Ampache based on a new framework.

73 From  AA1AD
Ernie D
Federal Way, WA | CN87ug

**Comment by Phyks** [comment]
@nasariasl Any news on this?

Also, I think it may be somehow linked to https://github.com/ampache/ampache/issues/1271.

**Comment by AshotN** [comment]
I could not reproduce. The only problem I had was when I clicked on the edit cog, it would load something but nothing would happen. Not until I clicked a second time.

But I was able to rename the title.

**Comment by AshotN** [comment]
I fixed the problem I was having, but I'm such a noob at Git that I've messed some stuff up. Once I fix my Git stuff, I'll push a fix.

**Comment by mitchray** [comment]
Closing due to inactivity, reopen if this is still relevant in latest version
